atlp-rwanda/ecommerce-app-legends-fe
# - seller two factor authentication #54

#### What does this PR do?

-  allow vendor to login completely after checking OTP
-  creation of view which support vendor through out that process

#### Description of Task which olready completed?

-  creating an input component
-  create a button component
-  create two-factor-view and use those components there
-  handle OTP submission 
-  return error and success messages

#### How should this be manually tested?
-  clone the repository https://github.com/atlp-rwanda/ecommerce-app-legends-fe
-   switch to ft-seller-2factor-Auth-#54 branch
-  run npm i
-  login as Vendor
-  if your creds are correct you gonna be redirected to this route `http://localhost:5173/verify-otp`
-  Enter OTP sent by the app to the email at the moment you were loggin in space provided on the app
-  the following is the image of the page you going to be redirected to
- <img width="1407" alt="Screen Shot 2023-04-30 at 05 43 23" src="https://user-images.githubusercontent.com/103331579/235352453-205b9481-eb42-4c48-b92c-37035e79d779.png">
